packages/web-components/fast-components/src/radio-group/radio-group.ts

```diff
@@ -0,0 +1,217 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import {
+    keyCodeArrowDown,
+    keyCodeArrowLeft,
+    keyCodeArrowRight,
+    keyCodeArrowUp,
+    keyCodeEnter,
+} from "@microsoft/fast-web-utilities";
+import { RadioControl } from "../radio";
+
+export class RadioGroup extends FASTElement {
+    @attr({ attribute: "readonly", mode: "boolean" })
+    public readOnly: boolean;
+    private readOnlyChanged(): void {
+        const filteredRadios = this.getFilteredRadioButtons();
+        if (filteredRadios !== undefined) {
+            filteredRadios.forEach((radio: HTMLInputElement) => {
+                if (this.disabled) {
+                    radio.setAttribute("readonly", "");
+                } else {
+                    radio.removeAttribute("readonly");
+                }
+            });
+        }
+    }
+
+    @attr({ attribute: "disabled", mode: "boolean" })
+    public disabled: boolean;
+    private disabledChanged(): void {
+        const filteredRadios = this.getFilteredRadioButtons();
+        if (filteredRadios !== undefined) {
+            filteredRadios.forEach((radio: HTMLInputElement) => {
+                if (this.disabled) {
+                    radio.setAttribute("disabled", "");
+                } else {
+                    radio.removeAttribute("disabled");
+                }
+            });
+        }
+    }
+
+    @attr
+    public name: string;
+    protected nameChanged(): void {
+        this.getFilteredRadioButtons().forEach((radio: HTMLInputElement) => {
+            radio.setAttribute("name", this.name);
+        });
+    }
+
+    @attr
+    public value: string;
+
+    @observable slottedRadioButtons: RadioControl[];
+    private selectedRadio: RadioControl | null;
+
+    constructor() {
+        super();
+        this.addEventListener("keydown", this.keydownHandler);
```

Comment:
Is there a reason this is set up in the constructor whereas the "change" handler is set up in the `connectedCallback`?

Reply:
no, the change handler should be up there as well, originally the change handler was on the individual radios but that is no longer the case, will move